modules/Quiz/resources/quiz.js

```diff
@@ -294,6 +294,7 @@
 
         retake : function(){
             var _this = this;
+            $(".retake-btn").css("pointer-events", "none");
```

Comment:
@shalommeoded  why CSS and not off() ?

Reply:
Changed,
I just used the same method that was used in similar cases.
But they work pretty much the same while testing.